src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -15,34 +15,70 @@
 
     private int total = TOTAL_VALUE;
 
+    /*
+     * (non-Javadoc)
+     *
+     * @see edu.kis.vh.nursery.Collections#countIn(int)
+     */
+    @Override
     public void countIn(int in) {
         if (!isFull()) {
-            numbers[++total] = in;
+            numbers[total++] = in;

```

Comment:
Changing TOTAL_VALUE requires such fixes? It does not make sense. Incorrect fix.